src/routes/idm.js

```diff
@@ -57,6 +58,51 @@ module.exports = [
       }
     }
   },
+  {
+    method: 'POST',
+    path: '/idm/' + version + '/user/change-email-address/start',
+    handler: changeEmailController.startChangeEmailAddress,
+    options: {
+      description: 'Get verification code for email change and send to new email',
+      validate: {
+        payload: {
+          userId: Joi.number().positive().required(),
+          password: Joi.string().required()
+        }
+      }
+    }
+  },
+  {
+    method: 'PATCH',
+    path: '/idm/' + version + '/user/change-email-address/create-code',
+    handler: changeEmailController.createVerificationCode,
+    options: {
+      description: 'Get verification code for email change and send to new email',
+      validate: {
+        params: {
+          userId: Joi.number().positive().required(),
+          verificationId: Joi.number().guid().required(),
+          email: Joi.string().email().required(),
+          application: Joi.string().required()
```

Comment:
I think I would expect the payload here to include only: 
```
          verificationId: Joi.number().guid().required(),
          email: Joi.string().email().required(),
```
As the user and application can be found via the email change and users tables